src/elm/Main.elm

```diff
@@ -197,6 +198,7 @@ type Msg
     | GotUpdatedBalances (Result Http.Error (List Community.Balance))
     | GotShopViewerMsg ShopViewer.Msg
     | GotTransferScreenMsg Transfer.Msg
+    | ShowFeedback Feedback.Model
```

Comment:
Why not just make it a `GotPageMsg` message since we already have this generalised structure?